src/screens/ContactScreen/index.js

```diff
@@ -37,7 +38,9 @@ const ContactScreen = ({navigation}) => {
   const [selectedUsers, setSelectedUsers] = React.useState([]);
   const [isSearchMode, setIsSearchMode] = React.useState(false);
   const [isLoadingSearchResult, setIsLoadingSearchResult] = React.useState(false);
-  const {createSignChat, loadingCreateChat} = useCreateChat();
+  const {createSignChat, createAnonymousChat, loadingCreateChat} = useCreateChat();
+  const route = useRoute();
+  const {from: sourceScreen} = route.params;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `createAnonymousChat` function is added to the destructuring assignment from `useCreateChat()`. The `route` and `sourceScreen` variables are also declared. These changes are necessary for creating anonymous chats based on the source screen.
```diff
+ const {createSignChat, createAnonymousChat, loadingCreateChat} = useCreateChat();
+ const route = useRoute();
+ const {from: sourceScreen} = route.params;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->